packages/tbds-utilities/lib/inline-icon.scss

```diff
@@ -0,0 +1,25 @@
+$_icon-margin: .2em;
+$_visually-center-offset: -.1em;
+
+.tbds-inline-icon {
+  height: 1em;
```

Comment:
If `1em` is what defines the height/size of the icon, I think that should be customizable with a default variable.